src/main.rs

```diff
@@ -36,9 +36,9 @@ struct ActionIntegrate {
     mods: Vec<String>,
 }
 
-/// Integrate a profile
+/// Integrate a mod group
 #[derive(Parser, Debug)]
-struct ActionIntegrateProfile {
+struct ActionIntegrateModGroup {
```

Comment:
Hmm, I think this logic here is wrong, it should still be ActionIntegrateProfile right?

Reply:
Correct. Integrating a single mod group doesn't make sense.